### ISSUE TITLE
`disabled` option is no longer showed

### ISSUE THREAD

**Issue by ogonkov** [select2/select2#2253]
If i set first option as placeholder for `select`, i. e.
```html
<select class="form-control input-sm">
  <option disabled selected>Select filter</option>
  <option value="43">CPC filter</option>
  <option value="42">Defaults</option>
  <option value="45">Google filter</option>
</select>
```

After initialization `select2` draw itself with no option selected by default.

I expect behaviour like in plain select -- `option` with `selected` attribute showed. No matter disabled it or not.

Here is a [JSBin with example](http://jsbin.com/payap/1/).

**Comment by kevin-brown** [comment]
Should be fixed in 4.0.0.

**Comment by ogonkov** [comment]
Yep, work like a charm, thanks, @kevin-brown 

http://jsbin.com/litamepeca/1/
